settings.gradle

```diff
@@ -0,0 +1,10 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        jcenter() // Warning: this repository is going to shut down soon
```

Comment:
We should drop `JCenter` imo as it's already [deprecated since March 31st of 2021](https://developer.android.com/studio/build/jcenter-migration).